File: cpp/18c_705A_Hulk.cpp
#include<bits/stdc++.h>

int main(){
    std::string s1 = "I hate", s2 = "I love", ans = "";
    int n;
    std::cin>>n;
    for(int i = 1; i <= n; i++){
        if (i % 2 != 0){
            ans += s1 + " ";
        }
        else{
            ans += s2 + " ";
        }
        if(i != n){
            ans += "that ";
        }
    }
    ans += "it";
    std::cout<<ans;
    return 0;
}

File: python/18c_705A_Hulk.py
def main():
    s1, s2 = "I hate", "I love"
    n = int(input())
    ans = ""
    for i in range(0,n):
        if i % 2 == 0:
            ans += s1 + " "
        else:
            ans += s2 + " "
        if i != (n-1):
            ans += "that "
    ans += "it"
    print(ans)

if __name__ == "__main__":
    main()